### PR TITLE
Add unit tests for authenticated content setServerToken

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

### Changed

1.  Added unit tests

### CODE REVIEW

File: packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { serverSetToken } from './../lib/authenticate'
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js]: Hey @dylankelly Should we use same name for tests directory across the whole project? At this moment, all are using `test` which is same as vue.js and nuxt.js project. I feel `__tests__` introduce the inconsistency.

File: packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        }
+      }
+    })
+
+    serverSetToken(`authenticatedContent=${validToken};`, store)
+    expect(store.dispatch).toHaveBeenCalledWith('tideAuthenticatedContent/setAuthenticated', true)
+  })
+
+  it('should isAuth to false when an expired JWT is passed in cookie', () => {
+    MockDate.set('2022-07-15T06:19:17+00:00')
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js]: 2022 is not very far, the test will fail three years later.

File: packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js (hunk 1 above)
Comment (dylankelly) [reply packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js]: I would like to change this, i think unit tests should always be colocated with corresponding code. I will change this PR and make another which changes this across the project though.

File: packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js (hunk 2 above)
Comment (dylankelly) [reply packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js]: yeah i'll make it one thousand years, by then the earth will be dead anyway 😄

File: packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js (hunk 1 above)
Comment (tim-yao) [reply packages/ripple-nuxt-tide/modules/authenticated-content/__tests__/authenticate.test.js]: @dylankelly Just check incase you forgot this one...

**Pull request merged**